phpunit.xml.dist

```diff
@@ -3,28 +3,19 @@
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-    backupGlobals="false"
-    backupStaticAttributes="false"
     beStrictAboutChangesToGlobalState="true"
     beStrictAboutOutputDuringTests="true"
-    beStrictAboutTestsThatDoNotTestAnything="true"
```

Comment:
Why the drop here ?

Reply:
All removed attributes were set to default values.

Reply:
I was adding them here because the "default" were changed across multiple PHPUnit major versions and we were using quite a wide range of them

Reply:
We reduced the range of PHP versions we support, which means we are less likely to use several versions of PHPUnit at the same time, if ever. So I'd say it's fine.